hadoop-hdfs-project/hadoop-hdfs-rbf/pom.xml

```diff
@@ -114,6 +114,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
assertj is already exported in test scope for hadoop-common. I'd have expected it to be picked up.